library/osbuild/src/utility/filesystem.rs

```diff
@@ -0,0 +1,28 @@
+use std::fs;
+use std::io;
+use std::path::Path;
+
+use walkdir::WalkDir;
+
+/// Copy a tree from `src` to `dst`
+pub fn copy_tree<P: AsRef<Path>>(src: P, _dst: P) -> Result<(), io::Error> {
```

Comment:
It doesn't matter too much here but I think you really want

```suggestion
pub fn copy_tree(src: impl AsRef<Path>, _dst: impl AsRef<Path>) -> Result<(), io::Error> 
```

because as it is today the API signature constrains source and destination unnecessarily to the same type.

But if going the cap-std path, python actually does support directory-relative file descriptors - which we could pass down into Rust I think, though we'd probably want to re-open them via cap-std.

Reply:
That makes sense!